indykite/resource_authorization_policy.go

```diff
@@ -15,20 +15,18 @@
 package indykite
 
 import (
-	"fmt"
-
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/structure"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 	"github.com/indykite/jarvis-sdk-go/config"
 	configpb "github.com/indykite/jarvis-sdk-go/gen/indykite/config/v1beta1"
-	knowledge_graphv1beta1 "github.com/indykite/jarvis-sdk-go/gen/indykite/knowledge_graph/v1beta1"
-	"google.golang.org/protobuf/encoding/protojson"
 )
 
 const (
 	authorizationPolicyJSONConfigKey = "json_config"
```

Comment:
I don't know if it matter but the proto field is now called `json`

Reply:
Well, it doesn't matter from funcional point of view. But yes, definitelly better to call similarly.